### PR TITLE
Adding back support for _canShowFeedback

### PR DESCRIPTION
**Pull request by jose-campos**
At the moment if you set _canShowFeedback to false and don't include feedback content the component errors out given the lack of the feedback content. This should take care of that.

### CODE REVIEW

**Comment by oliverfoster** [comment]
This needs to be fixed in the core not in each component.
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/models/questionModel.js#L198-L269

**Comment by jose-campos** [comment]
Agree, working on those now, but the MCQ overwrites the `setupFeedback` function so it also needs to be updated there.

File: js/mcqModel.js [hunk 1]
```diff
@@ -126,7 +126,8 @@ define([
         },
 
         setupFeedback: function() {
-
+            if (this.has('_canShowFeedback') && this.get('_canShowFeedback') == false)
```
Comment (tomgreenfield) [inline js/mcqModel.js]: Please amend to:
```js
if (!this.has('_feedback')) return;
```

**Comment by moloko** [comment]
can this be closed?

**Comment by moloko** [comment]
oops wrong button! guess question is irrelevant now

**Pull request closed without merging**